src/events.js

```diff
@@ -2184,44 +2202,40 @@ export const populateBoxTable = async (page, filter, source) => {
     currRow.insertCell(5).innerHTML = `Received<span style="display:block;">(Yes/No)</span>`;
     currRow.insertCell(6).innerHTML = "Date Received";
     currRow.insertCell(7).innerHTML = "Condition";
-    currRow.insertCell(8).innerHTML = "Comments"
+    currRow.insertCell(8).innerHTML = "Comments";
 
-    let conversion = {
-        "712278213": "FedEx",
-        "149772928": "World Courier"
-    }
-    
-    for (let i = 0; i < pageStuff.data.length; i++) {
+    for (let i = 0; i < reportPageBoxData.data.length; i++) {
         rowCount = currTable.rows.length;
         currRow = currTable.insertRow(rowCount);
-        let currPage = convertToOldBox(pageStuff.data[i]);
-        let numTubes = 0;
-        let keys = Object.keys(currPage['bags']);
-        for (let j = 0; j < keys.length; j++) {
-            numTubes += currPage['bags'][keys[j]]['arrElements'].length;
-        }
-        const shippedDate = currPage['656548982'] ? retrieveDateFromIsoString(currPage['656548982']) : '';
-        const receivedDate = currPage['926457119'] ? retrieveDateFromIsoString(currPage['926457119']) : '';
-        const packagedCondition = currPage['238268405'] || '';
-
-        currRow.insertCell(0).innerHTML = currPage[conceptIds.shippingTrackingNumber] ?? '';
+        const currBox = convertToOldBox(reportPageBoxData.data[i]);
+        const trackingNumber = currBox[conceptIds.shippingTrackingNumber] ?? '';
+        const shippedDate = currBox[conceptIds.shippingShipDate] ? retrieveDateFromIsoString(currBox[conceptIds.shippingShipDate]) : '';
+        const receivedDate = currBox[conceptIds.siteShipmentDateReceived] ? retrieveDateFromIsoString(currBox[conceptIds.siteShipmentDateReceived]) : '';
+        const packagedCondition = currBox[conceptIds.packageCondition] || '';
+        const shippingLocation = conceptIdToSiteSpecificLocation[currBox[conceptIds.shippingLocation]];
+        const boxId = currBox[conceptIds.shippingBoxId];
+        const viewManifestButton = '<button type="button" class="button btn btn-info" id="reportsViewManifest' + i + '">View manifest</button>';
+        const isReceived = currBox[conceptIds.siteShipmentReceived] === conceptIds.yes ? 'Yes' : 'No';
+        const packageConditionValue = convertConceptIdToPackageCondition(packagedCondition, packageConditionConversion);
+        const packageComments = currBox[conceptIds.siteShipmentComments] || '';
+
+        currRow.insertCell(0).innerHTML = trackingNumber;
```

Comment:
`innerText` and `textContent` have better performance for texts.

Reply:
updated